config/config_files/env.sh

```diff
@@ -24,6 +24,9 @@ echo "  ENVIRONMENT_NAME: \"$value\"," >>./env-config.js
 value=$(printf '%s\n' "${SERVICE_NOW_CONFIGURATION_ITEM}")
 echo "  SERVICE_NOW_CONFIGURATION_ITEM: \"$value\"," >>./env-config.js
 
+value=$(printf '%s\n' "${API_SUBSCRIPTION_KEY}")
+echo "  API_SUBSCRIPTION_KEY: \"$value\"," >>./env-config.js
```

Comment:
Since premo is the only app that uses subscription keys it is preferable to just change the `.github/workflows/check_config.ts` github action file in premo

Reply:
fixed